README.md

```diff
@@ -1,8 +1,9 @@
-# Doctrine CouchDB Client
+# Doctrine CouchDB v2.0.0 Client
 
-[![Build Status](https://travis-ci.org/doctrine/couchdb-client.png?branch=master)](https://travis-ci.org/doctrine/couchdb-client)
+[![Build Status](https://travis-ci.org/robsonvn/couchdb-client.png?branch=dev)](https://travis-ci.org/robsonvn/couchdb-client)
```

Comment:
We should really change this to `relaxedws`.

Reply:
Yes, for sure.